recipes/acl/all/conanfile.py

```diff
@@ -0,0 +1,41 @@
+import os
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class AclConan(ConanFile):
+    name = "acl"
```

Comment:
I thought this was an https://en.wikipedia.org/wiki/Access-control_list 🤣 

perhaps

```suggestion
    name = "nfrechette-acl"
```